app/services/paypal_service/api/accounts.rb

```diff
@@ -0,0 +1,228 @@
+module PaypalService::API
+  PaypalAccountStore = PaypalService::Store::PaypalAccount
+
+  class Accounts
+
+    def initialize(permissions, merchant, logger = PaypalService::Logger.new)
+      @permissions = permissions
+      @merchant = merchant
+      @logger = logger
+    end
+
+    # The API implmenetation
+    #
+
+    ## POST /accounts/request
+
+    def request(body:)
+      # If a request for new account is made, delete old (not completely actived) accounts
+      # Future note: This may not be sufficient solution when we implement the option to
+      # switch PayPal account
+      delete(
```

Comment:
I don't like API operations calling directly each other. There's unnecessary ceremony wrapping results etc. when we are already inside the module. I'd prefer using Store directly.